src/config.inc

```diff
@@ -76,7 +76,7 @@ endif
 OSX_IDENTITY="Developer ID Application: Daniel Kroening"
 
 # Detailed version information
-CBMC_VERSION = 5.95.1
+CBMC_VERSION = 6.0.0-preview
```

Comment:
I think the major version also needs to be propagated to here - https://github.com/diffblue/cbmc/blob/eaa2cdd997249de233aeacfdef7d3adf55cb7c2c/src/goto-programs/write_goto_binary.h#L15C1-L15C2

Reply:
No, please don't tweak that without a reason. It's coincidental that CBMC is at version 5.x.x and that the goto binary version is 5.

Reply:
I'd heard it said that they were supposed to coincide. Thank you for clarifying.